azure/sql-server/README.md

```diff
@@ -0,0 +1,242 @@
+
+        The name of an Azure resource group where the server will be deployed
+
+    - (string) **`location = null`** _[since v0.0.1]_
+
+        The name of an Azure location where the server will be deployed. If unspecified, the resource group's location will be used.
+
+- (string) **`name`** _[since v0.0.1]_
+
+    The name of the SQL server
+
+### Optional
+
+- (map(string)) **`additional_tags = {}`** _[since v0.0.1]_
+
+    Additional tags for the SQL server
+
+- (map(string)) **`additional_tags_all = {}`** _[since v0.0.1]_
+
+    Additional tags for all resources deployed with this module
+
+- (object) **`azuread_authentication = null`** _[since v0.0.1]_
+
+    Defines an Azure AD identity as administrator for this server. Can be used with `sql_authentication`
+
+    - (string) **`object_id`** _[since v0.0.1]_
+
+        The object ID of an Azure AD identity (user, group)
+
+    - (string) **`tenant_id = null`** _[since v0.0.1]_
+
+        The tenant ID for the domain where the identity lives
+
+- (string) **`connection_policy = "Default"`** _[since v0.0.1]_
+
+    The connection policy the server will use. Possible values are `"Default"`, `"Proxy"`, and `"Redirect"`
+
+- (map(object)) **`databases`** _[since v0.0.1]_
+
+    Configures and manages multiple databases that are attached to this server
+
+    - (map(string)) **`additional_tags = {}`** _[since v0.0.1]_
+
+        Additional tags for the database
+
+    - (string) **`backup_storage_redundancy = "Geo"`** _[since v0.0.1]_
+
+        Specifies the storage account type used to store backups for this database. Possible values are `"Geo"`, `"Local"` and `"Zone"`
+
+    - (bool) **`bring_your_own_license = false`** _[since v0.0.1]_
+
+        Use your license you already own with Azure Hybrid Benefit
+
+    - (string) **`collation = "SQL_Latin1_General_CP1_CI_AS"`** _[since v0.0.1]_
+
+        Database collation defines the rules that sort and compare data, and cannot be changed after database creation
+
+    - (string) **`create_mode = "Default""`** _[since v0.0.1]_
+
+        Defines the create action of the database. Possible values are `"Copy"`, `"Default"`, `"OnlineSecondary"`, `"PointInTimeRestore"`, `"Recovery"`, `"Restore"`, `"RestoreExternalBackup"`, `"RestoreExternalBackupSecondary"`, `"RestoreLongTermRetentionBackup"` and `"Secondary"`
+
+    - (number) **`data_max_size = null`** _[since v0.0.1]_
+
+        The max size of the database in gigabytes.        
+
+    - (object) **`dtu_model = null`** _[since v0.0.1]_
+
+        Configures the database using the DTU pricing model
+
+        - (string) **`tier`** _[since v0.0.1]_
+
+            Defines the tier of this database. Possible values are `"Basic"`, `"Standard"`, and `"Premium"`. Note that some tiers are not available for some regions. Run this CLI command to get a list of tiers applicable to your region. `az sql db list-editions --location westus --output table`
```

Comment:
From reading this, I'm to assume that `--location westus` needs to be changed based on a person's need? If so, I'll rephrase this sentence a bit.

Reply:
clarification added.